src/components/Star/Star.vue

```diff
@@ -1,39 +1,40 @@
 <template>
-    <div class="d-flex justify-content-between mb-lg pb-2">
-        <div class="text-dark">
-            <h2>
-                <i class="far fa-star text-warning mr-xs pointer" 
-                v-for="starKey in displayStar" :key="starKey" 
-                @click="rating(starKey)"
-                v-bind:class="{ 'fa' : starKey <=  ratedStar}"
-                :style="{'font-size' : `${starSize}`}"
-                />
-            </h2>
-        </div>
+  <div class="d-flex justify-content-between mb-lg pb-2">
+    <div class="text-dark">
+      <h2>
+        <i
+          :disable="forDaisable === true ? true : false"
```

Comment:
*forDaisable* there is no need to use ternary operator, remove it, it doesn't make sense here

Reply:
removed ternary operator